src/assets/simp-environment/environments/simp/hieradata/simp_classes.yaml

```diff
@@ -11,68 +11,62 @@
 # possible.                                                          #
 #                                                                    #
 # For more information on how to use Hiera, see:                     #
-#   * http://docs.puppetlabs.com/hiera/1/puppet.html                 #
-#   * http://docs.puppetlabs.com/hiera/1/complete_example.html       #
+#   * http://docs.puppetlabs.com/hiera/latest/puppet.html            #
+#   * http://docs.puppetlabs.com/hiera/latest/complete_example.html  #
 #                                                                    #
 ######################################################################
 
 classes:
-  - 'aide'
-  - 'auditd'
+  - '::aide'
```

Comment:
Do these work scoped in Hiera? Also, Puppet 4 shouldn't need to be scoped any more but we still do it in the modules for clarity.